iceoryx_hoofs/include/iceoryx_hoofs/iceoryx_hoofs_types.hpp

```diff
@@ -46,6 +48,9 @@ enum class LogLevel : uint8_t
 /// @brief converts LogLevel into a string literal
 /// @param[in] value the LogLevel to convert
 /// @return string literal of the LogLevel value
+// AXIVION Next Construct AutosarC++19_03-A3.9.1 : Requires char base type to print strings originating
+// from string literals, cxx::string::c_str(), std::string::c_str() or other sources. Only with char we
+// can achieve the task.
```

Comment:
A better justification in this case would be `This function return a string literal which corresponds to a const char *`